src/components/Map.js

```diff
@@ -1,8 +1,116 @@
 import React, { Component } from 'react';
 import { connect } from 'react-redux';
+import './styling/Map.css';
 
-class map extends Component {
+import {GoogleMap, withScriptjs, withGoogleMap} from "react-google-maps";
+import restaurantsData from '../restaurant.data';
+import Card from 'react-bootstrap/Card'
+import Col from "react-bootstrap/Col";
+import Row from "react-bootstrap/Row";
+import Badge from "react-bootstrap/Badge";
+import Image from "react-bootstrap/Image";
+import StarRatings from 'react-star-ratings';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { faSortDown, faSortUp } from '@fortawesome/free-solid-svg-icons'
+import { Button } from 'react-bootstrap';
+
+class Map extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      restaurants: restaurantsData,
+    }
+  }
+  onSortUp() {
+    let sorted = [...this.state.restaurants].sort((a, b) => {
+      return a.rating - b.rating;
+    });
+    this.setState({
+      restaurants: sorted
+    });
+  }
+  onSortDown() {
+    let sorted = [...this.state.restaurants].sort((a, b) => {
+      return b.rating - a.rating;
+    });
+    this.setState({
+      restaurants: sorted
+    });
+  }
+
+  Map() {
+    return <GoogleMap defaultZoom={15} defaultCenter={{lat: 43.2609, lng: -79.9192}} />
+  }
+
+  WrappedMap = withScriptjs(withGoogleMap(this.Map));
+
+  render() {
+    return (
+      <div className='map-container'>
+        <div style={{ width: "50vw", height: "100vh" }} className='map'>
+          <this.WrappedMap googleMapURL="https://maps.googleapis.com/maps/api/js?v=3.exp&libraries=geometry,drawing,places"
+          loadingElement={<div style={{ height: `100%` }} />}
+          containerElement={<div style={{ height: `100%` }} />}
+          mapElement={<div style={{ height: `100%` }} />} />
+        </div>
+        <div className='category'>
+          <Card border="primary">
+            <Card.Header style={{ "font-size": "1.8em" }}>All Restaurants</Card.Header>
+            <Card.Body>
+              <Card.Text style={{ "font-size": "1.5em" }}>
+                Below are a list of all restaurants. All of them would have been displayed on Google Map, but due to development budgets, we are unable to mark them on map. 
```

Comment:
We need to have this implemented. Can't we use something similar to what Dr. Browne did using Bing Maps?

Reply:
i also looked into bing map as well, but as far as im concerned, that also requires api key keys. but i think right now i would be a good placeholder until further investigation can be done. my hands are tied between now and saturday. ill look into the map on saturday, but it would be great if you want to help as well. But right now, just want to get the map showing in the right place for this pr